src/assets/css/layouts/header.css

```diff
@@ -32,4 +32,8 @@
 .navbar-link {
   padding-left: 30px;
   font-weight: var(--fw-regular);
+}
+.navbar-button a{
```

Comment:
- Code style
- As I told you before, we shouldn't style directly HTML elements. Please check all files and update if have

Reply:
Thank you, i fixed it